mqtt/mqtt-broker/src/connection/mod.rs

```diff
@@ -158,7 +158,7 @@ where
                     Ok(Some(auth_id)) => Auth::Identity(auth_id),
                     Ok(None) => Auth::Unknown,
                     Err(e) => {
-                        warn!(message = "error authenticating client", error =% *e);
+                        warn!(message = "error authenticating client", error =% &*e);
```

Comment:
So they changed the behavior in a minor version update?

Reply:
They did some refactoring in the internals, but it affected this particular case. When I make a smaller example I faced compiler error even without tracing. I wonder how it even worked before :)